addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -64,10 +64,9 @@ def _onchange_event_id_booth(self):
         if self.event_booth_pending_ids and (not self.event_id or self.event_id != self.event_booth_pending_ids.event_id):
             self.event_booth_pending_ids = None
 
-    @api.onchange('event_booth_pending_ids')
-    def _onchange_event_booth_pending_ids(self):
-        # trigger the description computation
-        self.product_id_change()
+    @api.depends('event_booth_pending_ids')
+    def _compute_name(self):
```

Comment:
As it is void, do we have to keep it ? Or has it some magic hidden somewhere ?

Reply:
Both have a custom override of `get_sale_order_line_multiline_description_sale` below.
I've updated the method docstring accordingly :).